ChangeLog.md

```diff
@@ -42,8 +42,7 @@
 
 ## 0.6.2.0
 
-* Fix `holdDyn` so that it is lazy in its event argument  
-  These produce `DMap`s  whose values needn't be `Identity`.
+* Add `Reflex.Query.Base.mapQueryT`. See that module for documentation
 * Stop using the now-deprecated `*Tag` classes (e.g., `ShowTag`).
 * Fix `holdDyn` so that it is lazy in its event argument.
```

Comment:
It was duplicated here, it seems

Reply:
Hmm only half of it though?

Reply:
We should fix that in another PR. In the meantime, I moved the new entry for this PR's changes to the top where it belongs.